### ISSUE TITLE
Limelight aiming

### ISSUE THREAD

**Issue by willitcode** [frc937/robot2024#146]
Limelight aiming doesn't work. Why? That part needs investigation.

Even once it does work, we should still tweak it—instead of the tx value translating into robot rotation, it should translate into strafing robot movement relative to the amp. Assuming we get heading snapping working, we can use heading snapping to lock the robot's heading into pointing straight towards the amp as well. Theoretically, this should lead to a very efficient aiming routine—assuming that the Limelight's limited FOV doesn't result in us immediately losing sight of the AprilTag. If that happens, we can maybe experiment with using the tx value as input for both strafe and rotation... but that could get weird fast.

**Comment by willitcode** [comment]
In do later because Heartland branch needs to merge before we can do anything & troubleshooting requires access to a bot that is not in several pieces

**Comment by willitcode** [comment]
Should theoretically have been fixed by #159. Still needs testing.

**Comment by willitcode** [comment]
idek what is going on with LL aiming anymore... tested a lot at comp, nothing seems to be working. Our best bet at this point may be to use heading snapping to lock ourselves towards the side that the amp is on and then use the limelight to drive forward, back, and side to side... but I'm not sure if that actually works. IIRC we implemented it at comp and it didn't really seem to. We're also having some oscillation issues with the speed multipliers, so we'll see how that goes.

TLDR: needs an absurd amount of testing and me just sitting there trying to figure it out. We probably don't have time for it.

**Comment by willitcode** [comment]
We should get rid of the logic that automatically ends the command if we lose sight of the target instead of just commenting it out (this is a todo)
